extensions/amp-story/1.0/amp-story-interactive-results-category.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpStoryInteractive, InteractiveType} from './amp-story-interactive';
+import {CSS} from '../../../build/amp-story-interactive-results-category-1.0.css';
+import {StateProperty} from './amp-story-store-service';
+import {htmlFor} from '../../../src/static-template';
+import {setStyle} from '../../../src/style';
+
+/**
+ * Generates the template for the quiz.
+ *
+ * @param {!Element} element
+ * @return {!Element}
+ */
+const buildResultsCategoryTemplate = (element) => {
+  const html = htmlFor(element);
+  return html`
+    <div class="i-amphtml-story-interactive-results-category-container">
+      <div class="i-amphtml-story-interactive-results-pre-select">
+        <span>Come back when you're done</span>
+      </div>
```

Comment:
Pre-finished state is not defined yet, so this PR has just a div that says "come back when you're done". Will be updated with the correct layout and styles in a future PR